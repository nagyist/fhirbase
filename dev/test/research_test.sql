--db:fhirb -e
SET escape_string_warning=off;
--{{{

SELECT
  assert_eq(12::bigint, count(*), 'split params')
  FROM _query_string_to_params(
   'provider._id=1,2&provider.name=ups&_id=1,2&birthdate:missing=true&identifier=MRN|7777777&_count=100&name=pups&_sort:desc=name&_sort:asc=address&_page=10&_tag=category&_security=ups'
  ) _;
--}}}
