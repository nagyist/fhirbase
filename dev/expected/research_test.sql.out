SET escape_string_warning=off;
SET
SELECT
  assert_eq(12::bigint, count(*), 'split params')
  FROM _query_string_to_params(
   'provider._id=1,2&provider.name=ups&_id=1,2&birthdate:missing=true&identifier=MRN|7777777&_count=100&name=pups&_sort:desc=name&_sort:asc=address&_page=10&_tag=category&_security=ups'
  ) _;
    assert_eq    
-----------------
 OK split params
(1 row)

